client/deposits/filters/config.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export const filters = [
+	{
+		label: __( 'Deposit currency', 'woocommerce-payments' ),
+		param: 'store_currency_is',
+		staticParams: [ 'paged', 'per_page', 'orderby', 'order' ],
```

Comment:
The designs for this page include advanced filters. Should we create a new issue to track adding those? Just to make it clear we're not planning to do it as part of this multi-deposit currency work.

Reply:
Yes, we definitely need a ticket for the advanced filters. Will create a new one on Monday.

Reply:
https://github.com/Automattic/woocommerce-payments/issues/1545